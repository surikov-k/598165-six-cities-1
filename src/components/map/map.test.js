import React from 'react';
import renderer from 'react-test-renderer';
import mockLeaflet from '../../mocks/mock-leaflet.js';

import Map from './map.jsx';

const mock = [
  {
    id: 0,
    imgSrc: `img/apartment-01.jpg`,
    price: 120,
    rating: 4,
    name: `Canal View Prinsengracht`,
    type: `apartment`,
    premium: true,
    origin: [0, 0],
  },
];
const div = global.document.createElement(`div`);
div.setAttribute(`id`, `map`);
global.document.body.appendChild(div);

describe(`Map component`, () => {
  it(`renders properly`, () => {
    const tree = renderer.create(
        <Map
          offers={mock}
          leaflet={mockLeaflet}
        />

    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
