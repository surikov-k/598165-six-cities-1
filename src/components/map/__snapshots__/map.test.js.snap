// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map component renders properly 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "823px",
    }
  }
/>
`;
